local/local_provider_test.go

```diff
@@ -154,7 +154,7 @@ func TestUpdateDatabase(t *testing.T) {
 			Resource: "persistentvolumeclaims",
 		},
 		{
-			Action:   "update",
```

Comment:
please don't remove a test :)

Reply:
not a test, a stage of the resource interaction